docs/guides/operator/installation.adoc

```diff
@@ -44,6 +44,65 @@ image::configure-operator.png["Configure {project_name} Operator"]
 
 You may select to either have the Operator watch the namespace where it is installed, or to watch a single namespace of your choosing.
 
+==== Configuring Manual Approval for OLM Upgrades
+
+[WARNING]
+====
+*Important: Automatic OLM Upgrades*
+
+By default, OLM automatically updates the {project_name} Operator when a new version is released. This can cause several significant issues:
+
+*Major version upgrades* present the highest risk:
+
+* When using the default {project_name} image, the Operator uses a matching image of the corresponding {project_name} version, resulting in *unintended {project_name} upgrades* when the Operator is upgraded
+* *Even when using custom images*, major Operator upgrades can introduce significant compatibility issues with your existing Keycloak CR configuration, potentially requiring manual intervention
+* No option to downgrade to the previous {project_name} version due to database schema migrations
+* Breaking changes in the Operator's handling of Custom Resources
```

Comment:
I don't think we need this since we already have:
```
*Even when using custom images*, major Operator upgrades can introduce significant compatibility issues with your existing Keycloak CR configuration, potentially requiring manual intervention
```

Reply:
Agree. I will remove it.